test/mixer_test.py

```diff
@@ -1059,9 +1060,11 @@ def todo_test_sound__from_buffer(self):
         """Ensure Sound() creation with a buffer works."""
         self.fail()
 
-    def todo_test_sound__from_array(self):
+    def test_sound__from_array(self):
         """Ensure Sound() creation with an array works."""
-        self.fail()
+        array1 = sndarray.array(example_path(os.path.join("data", "house_lo.wav")))
```

Comment:
I wonder if this should be using pythons array.array rather than the pygame.sndarray? This way it would not depend on sndarray, but only what is in the python standard library.